### ISSUE TITLE
Full screen snapshot images have not alt text or accessible labels

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-org#2968]
When generating screenshots for full screen demos or examples, we need to also generate an accessible label as alt text - or point to the example/demo heading as it's label.

This will fix MANY of the website a11y issues.

**Comment by evwilkin** [comment]
Full screen `<img>` is generated in [example.js](https://github.com/patternfly/patternfly-org/blob/main/packages/theme-patternfly-org/components/example/example.js#L175) and should be able to reference the `title` prop used in the anchor tag wrapping the image.
